libcxx/test/benchmarks/vector_operations.bench.cpp

```diff
@@ -14,6 +14,7 @@
 #include <deque>
 #include <functional>
 #include <vector>
+#include <memory>
```

Comment:
Currently this file also uses `std::string` without explictly including `<string>`. Perhaps we should add `#include <string>` together.

Reply:
Thank you for your reminder. I'll also add that.